src/main/java/com/opentable/versionedconfig/GitProperties.java

```diff
@@ -35,15 +37,28 @@ public GitProperties(URI remoteRepository,
                          @Nullable String password,
                          @Nullable File localRepository,
                          String branch) {
-        this.remoteRepository = remoteRepository;
+        this(Collections.singletonList(remoteRepository), username, password, localRepository, branch);
+    }
+
+    public GitProperties(List<URI> remoteRepositories,
+                         @Nullable String username,
+                         @Nullable String password,
+                         @Nullable File localRepository,
+                         String branch) {
+        this.remoteRepositories = remoteRepositories;
         this.username = username;
         this.password = password;
         this.localRepository = localRepository;
         this.branch = branch;
     }
 
+    @Deprecated
```

Comment:
Do we need to maintain compatibility here?

Reply:
honestly probably not.  I was just being technically correct.  I'll remove it if you think we should break compatibility.